modules/exploits/multi/http/papercut_ng_auth_bypass.rb

```diff
@@ -0,0 +1,254 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'cgi'
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+  prepend Msf::Exploit::Remote::AutoCheck
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::Remote::HttpServer
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'PaperCut PaperCutNG Authentication Bypass',
+        'Description' => %q{
+          This module leverages an authentication bypass in PaperCut NG. If necessary it
+          updates Papercut configuration options, specifically the 'print-and-device.script.enabled'
+          and 'print.script.sandboxed' options to allow for arbitrary code execution running in
+          the builtin RhinoJS engine.
+
+          This module logs at most 2 events in the application log of papercut. Each event is tied
+          to modifcation of server settings.
+        },
+        'License' => MSF_LICENSE,
+        'Author' => ['catatonicprime'],
+        'References' => [
+          ['CVE', '2023–27350'],
```

Comment:
This weird hyphen character also makes the linter fail:
```suggestion
          ['CVE', '2023-27350'],
```